### DISCUSSION TITLE
Unable to do Automatic Speech Recognition / Speech-to-Text over more than 30 sec of the audio files

### DISCUSSION THREAD

**Discussion by PoissonChasseur** [speechbrain/speechbrain#2149] [Q&A]
### Describe the bug

I'm quite new to "Automatic Speech Recognition / Speech-to-Text" and I just started yesterday using SpeechBrain" which I installed yesterday with the latest version available for PyTorch 1.X (speechbrain 0.5. 15, PyTorch 1.13.1, cuda 11.7, torchaudio 0.13.1, soundfile 0.12.1).

I followed the tutorial on ["Speech Recognition"](https://colab.research.google.com/drive/1aFgzrUv3udM_gNJNUoLaHIm78QHtxdIz?usp=sharing#scrollTo=uo0JP7a5uFp7), in order to use pre-trained models on some audio files that have been individually tested with different pre-trained AI models. These audio files were sampled at 44.1KHz with durations of 32sec, 36sec, 1m54, 2m02.

**However, strangely, the convesion of the audio file to text seems to stop a bit before I've done 30 seconds of the audio file (seems to stop around about 27-29 seconds).** Can I do something to ensure that the whole audio file is converted to text, instead of only partially converted?

### Expected behaviour

Having the results of the "speech to text" conversion over the entire audio file instead of only being done over about 27-30 seconds of the audio files.

### To Reproduce

This is the Python code I was using:
```
import numpy as np, torch, soundfile, torchaudio, speechbrain as sb

device = "cuda:0" if torch.cuda.is_available() else "cpu";

audio_file = "[Name of the audio file to use].wav"; # Those used were in 44.1KHz (instead of 16KHz or 8KHz).
assert os.path.isfile(audio_file), "ERROR: Invalid audio_file:\n\t%s" % (audio_file);

#-----------------------------------------------------
from speechbrain.pretrained import EncoderDecoderASR

run_opts  = {"device": device};
source = "speechbrain/asr-crdnn-rnnlm-librispeech"; 
asr_model = EncoderDecoderASR.from_hparams(source=source, savedir="/content/pretrained_model", run_opts = run_opts);

result = asr_model.transcribe_file(audio_file);

print("\nresult:");
print(result);
print("\n");
```

### Versions

speechbrain 0.5.15, PyTorch 1.13.1, cuda 11.7, torchaudio 0.13.1, soundfile 0.12.1

**Comment by Adel-Moumen** [comment]
Hello @PoissonChasseur,

The model that you are using is trained on 16kHz. 

Could you please do some downsampling to obtain the same frequency? 
Let me know of your results. 

best, 
Adel
